packages/ui-breadcrumb/src/Breadcrumb/BreadcrumbLink/props.ts

```diff
@@ -29,13 +29,32 @@ import type {
   OtherHTMLAttributes,
   PropValidators
 } from '@instructure/shared-types'
+import type { ViewOwnProps } from '@instructure/ui-view'
 
 type BreadcrumbLinkOwnProps = {
+  /**
+   * Content to render as the crumb, generally should be text.
+   */
   children: React.ReactNode
+  /**
+   * Link the crumb should direct to; if an href is provided, the crumb will render as a link
+   */
   href?: string
-  onClick?: (...args: any[]) => any
+  /**
+   * If the Breadcrumb.Link has an onClick prop (and no href), it will render as a button
+   */
+  onClick?: (event: React.MouseEvent<ViewOwnProps, MouseEvent>) => void
+  /**
+   * Sets the font-size of the breadcrumb text
+   */
   size?: 'small' | 'medium' | 'large'
-  renderIcon?: React.ReactNode | ((...args: any[]) => any)
+  /**
+   * Add an icon to the Breadcrumb.Link
+   */
+  renderIcon?: React.ReactNode | (() => React.ReactNode)
+  /**
+   * Place the icon before or after the text in the Breadcrumb.Link
+   */
   iconPlacement?: 'start' | 'end'
 }
```

Comment:
I see this in the test:
```
    //@ts-expect-error fix this, but how? `to` is not a prop BreadcrumbLink
    await mount(<BreadcrumbLink to="/example">Test</BreadcrumbLink>)
```

When I check `Link` (where the props are forwarded) there is this explanation:

```
type LinkProps = LinkOwnProps &
  WithStyleProps<LinkTheme, LinkStyle> &
  OtherHTMLAttributes<LinkOwnProps> & {
    // React Router might add a `to` prop
    to?: string
  }
```

This means to me, that we should allow this `to` prop in `BreadcrumbLink` too.

Reply:
Yeah I just commented the same thing :)